tools/ci/ethereum_test

```diff
@@ -11,22 +11,24 @@ heading 'Setup...'
 echo 'logging in..'
 # we do this at run-time instead of build time
 # to eliminate the risk of copying over a stale chainlink binary
-# if we had instead copied over the binary from the built chainlink 
+# if we had instead copied over the binary from the built chainlink
 # base image
 # we could also try mounting a volume instead, or using docker-exec
 docker cp chainlink-node:/usr/local/bin/chainlink /usr/local/bin/chainlink
 chainlink admin login --file /run/secrets/apicredentials
 launch_chainlink
 deploy_contracts
+deploy_v05_contracts
```

Comment:
This might clobber from environment variables under the same name, perhaps move it to line 28? Those are the tests that need v05 contracts.